Source/WebCore/page/LocalFrameViewLayoutContext.h

```diff
@@ -128,6 +129,10 @@ class LocalFrameViewLayoutContext {
     UpdateScrollInfoAfterLayoutTransaction& updateScrollInfoAfterLayoutTransaction();
     UpdateScrollInfoAfterLayoutTransaction* updateScrollInfoAfterLayoutTransactionIfExists() { return m_updateScrollInfoAfterLayoutTransaction.get(); }
 
+    void markNeedingTransformUpdate(RenderLayerModelObject* parent, RenderLayerModelObject* descendant);
+    Vector<WeakPtr<RenderLayerModelObject>> descendantsNeedingTransformUpdate(RenderLayerModelObject* parent);
+    void markFinishedLayout(RenderLayerModelObject* parent);
```

Comment:
Not a fan of `markFinishedLayout` (layout is not really finished for this block). It's more like `hasFinalSize()` or something.

Reply:
Changed to finishedUpdatingDescendants.